frontend/src/App.js

```diff
@@ -50,18 +50,16 @@ function App() {
           [
               {
                   command: command,
-                  args:
-                  [
-                      group: group,
-                      state: state 
+                  args:[
+                    args
```

Comment:
This takes an array (`args`) and puts it inside another array. It should just be `args: args`